diff --git a/python/hostconfig/macosx.py b/python/hostconfig/macosx.py
index f1671d8..b72ba6f 100644
--- a/python/hostconfig/macosx.py
+++ b/python/hostconfig/macosx.py
@@ -1,7 +1,3 @@
-# This configuration is for Mac OSX Mountain Lion
-# with package installed via Homebrew
-# See https://chaste.cs.ox.ac.uk/trac/wiki/InstallGuides/ChasteInstallationOnMountainLion
-
 """Copyright (c) 2005-2015, University of Oxford.
 All rights reserved.
 
@@ -36,26 +32,18 @@ OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
 import os
 
-petsc_ver = 3.3
-petsc_path='/usr/local/Cellar/petsc/3.3-p5/'
+petsc_ver = 3.6
+petsc_path='/usr/local/Cellar/petsc/3.6.1_4'
 petsc_build_name = ''
 petsc_build_name_profile = ''
 petsc_build_name_optimized = ''
 
 noccache = "true"
 
-other_includepaths = ['/usr/local/opt/libxsd',
-                      '/usr/local/include/']
-
-other_libpaths = [ '/usr/X11/lib',
-                  '/usr/local/lib/']
+other_includepaths = ['/usr/local/include/xsd']
+other_libpaths = []
 blas_lapack = []
-other_libraries = ['X11', 'boost_serialization-mt', 'boost_filesystem-mt', 'boost_system-mt','xerces-c', 'z', 'hdf5', 'parmetis', 'metis']
-
-
-# Location of Apple's versions of BLAS and LAPACK: -framework vecLib
-ldflags='-framework vecLib'
-
+other_libraries = ['boost_serialization-mt', 'boost_filesystem-mt', 'boost_system-mt','xerces-c', 'z', 'hdf5', 'parmetis','metis']
 
 def Configure(prefs, build):
     """Set up the build configuring.
@@ -69,9 +57,9 @@ def Configure(prefs, build):
     global use_vtk
     use_vtk = True
     if use_vtk:
-        other_includepaths.append('/usr/local/include/vtk-5.10')
-        other_libpaths.append('/usr/local/lib/vtk-5.10')
-        other_libraries.extend(['vtkFiltering', 'vtkIO', 'vtkCommon', 'vtksys', 'vtkzlib', 'vtkexpat', 'vtkGraphics'])
+        other_includepaths.append('/usr/local/opt/vtk5/include/vtk-5.10')
+        other_libpaths.append('/usr/local/opt/vtk5/lib/vtk-5.10')
+        other_libraries.extend(['vtkFiltering', 'vtkIO', 'vtkCommon', 'vtksys', 'vtkGraphics'])
 
     # CVODE setup
     global use_cvode
